### ISSUE TITLE
sitewide, design system - 508-defect-3 [COGNITION, MOBILE DESIGN]: Styling SHOULD support reading comprehension

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#12680]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

```diff
! Team affected: VSA BAM1; Project: BDD
```

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

In the mobile view, the line-height of the `x` is `37px`, and makes it difficult to read that the individual lines are connected. Accessibility best practices recommends a line-height **should** be 1.5. It's great that this meets the minimum of 1.5, and this expanded line-height is likely to be a reading comprehension issue for some with cognitive considerations.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a mobile device user, I want to read the content of a document and comprehend text paragraphs.

## Environment

* Operating System: all — **mobile device layout**
* Browser: any
* Server destination: staging (and sitewide)

## Steps to Recreate

1. Logged in as user 261, enter https://staging.va.gov/disability/file-disability-claim-form-21-526ez/review-and-submit in browser
2. This should be on the Review screen
3. Using developer tools, resize the layout to a mobile device (for example, 320x480)
4. Verify that the heading for "Answer online questions or upload paper 21-0781A?" has a line-height that may interfere with reading comprehension

## Possible Fixes (optional)

```diff
<h3 class="form-review-panel-page-header vads-u-font-size--h5">
  Answer online questions or upload paper 21-0781A?
</h3>

<style>
.form-review-panel-page-header {
    flex: 1;
    padding-bottom: 0!important;
    padding-right: 5px;
!    line-height: 37px;
+    line-height: 1.5;
    margin-top: 0;
    margin-bottom: 0;
}
</style>
```

## WCAG or Vendor Guidance (optional)

* [W3C WCAG 2.0 - Success Criterion 1.4.12 Text Spacing (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html) - In content implemented using markup languages that support the following text style properties, no loss of content or functionality occurs by setting all of the following and by changing no other style property:
    - Line height (line spacing) to at least 1.5 times the font size;
    - Spacing following paragraphs to at least 2 times the font size;
    - Letter spacing (tracking) to at least 0.12 times the font size;
    - Word spacing to at least 0.16 times the font size.

* [W3C Understanding WCAG 2.0 - C21: Specifying line spacing in CSS](https://www.w3.org/TR/WCAG20-TECHS/C21.html) - Many people with cognitive disabilities have trouble tracking lines of text when a block of text is single spaced. Providing spacing between 1.5 to 2 allows them to start a new line more easily once they have finished the previous one.

### Current 

![Screen Shot 2020-08-20 at 2 38 00 PM](https://user-images.githubusercontent.com/57469/90811533-c5fee200-e2f2-11ea-812e-cdbf5abb1955.png)

### Recommendation

![Screen Shot 2020-08-20 at 2 37 11 PM](https://user-images.githubusercontent.com/57469/90811489-b2ec1200-e2f2-11ea-9425-f19324e4848d.png)

**Comment by laflannery** [comment]
@jstrothman We found this in our backlog and it relates to the 526ez form rather than Sitewide. It very well may resolved as it's from a while ago but we wanted to make sure it was sent to the right person rather than just closing it, thanks!

**Comment by jstrothman** [comment]
@andaleliz Would you want to take a look at this and see if it's still an issue? The issue is quite old (2020) and I don't have any additional background on it.

**Comment by andaleliz** [comment]
Yep, I'll take a look this week! Thanks for flagging

**Comment by andaleliz** [comment]
I'm going to close this - looks like the line height has been fixed! I don't love the bottom margin on the sections, but that's a different problem ;) 

<img width="1389" alt="Image" src="https://github.com/user-attachments/assets/495fba02-1942-49c1-87b4-c76ebfab9e48" />
